services/QuillLMS/app/controllers/teachers/classroom_manager_controller.rb

```diff
@@ -75,6 +75,8 @@ def dashboard
     @objective_checklist = generate_onboarding_checklist
     @first_name = current_user.first_name
 
+    growth_diagnostic_promotion_card_milestone = Milestone.find_by_name(Milestone::TYPES[:acknowledge_growth_diagnostic_promotion_card])
+    @show_diagnostic_promotion_card = !UserMilestone.find_by(milestone_id: growth_diagnostic_promotion_card_milestone&.id, user_id: current_user&.id) && (current_user.created_at < "2021-11-29".to_date || current_user&.unit_activities&.where(activity_id: Activity.diagnostic_activity_ids)&.any?)
```

Comment:
I presume that adding a check box to the dashboard onboarding experience is the reason we only feel like we need to show the banner to "old" users?  We expect new users to get caught by the updated onboarding flow?

Reply:
This doesn't add a checkbox to the user onboarding experience - those are in the `Objective` table. I think the idea is just that it doesn't really make sense for users to see it who aren't either a) already acquainted with the website or b) have already assigned a diagnostic, because we don't want to push them to assign a growth diagnostic right off the bat (we want them to assign a baseline diagnostic and then follow-up with a growth later).

Reply:
Is there any thought to changing this date as time moves forward, then?  Or is this banner a one-time feature that, at some point, we'll probably want to come back and remove?

Reply:
I think it'll eventually be removed.